### PR TITLE
[2.11_stage] Fix OCP install links

### PR DESCRIPTION
**Pull request by dhaiducek**
Cherry-pick:
- #7015

### CODE REVIEW

**Review by swopebe — APPROVED** [review]

File: install/cluster_size.adoc [hunk 1]
```diff
@@ -86,7 +86,7 @@ Also learn more about link:https://cloud.google.com/compute/docs/machine-types[m
 | 4 (2 cores per socket)
 | 16 GB
 | 120 GB
-| See the following link:https://docs.openshift.com/container-platform/4.6/installing/installing_vsphere/installing-vsphere-installer-provisioned.html[product documentation] for more details.
+| See the following link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/installing_on_vsphere[Installing on vSphere] in the {ocp-short} documentation for more details.
```
Comment (cadawson) [inline install/cluster_size.adoc:89]:
```suggestion
| See link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/installing_on_vsphere[Installing on vSphere] in the {ocp-short} documentation for more details.
```

File: install/cluster_size.adoc [hunk 2]
```diff
@@ -128,15 +128,15 @@ For more information about SMT, see link:https://www.ibm.com/docs/en/aix/7.2?top
 | 4
 | 16 GB
 | 120 GB
-| See the following link:https://docs.openshift.com/container-platform/4.14/installing/installing_bare_metal/installing-restricted-networks-bare-metal.html#installation-three-node-cluster_installing-restricted-networks-bare-metal[product documentation] for more details.
+| See the following link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/disconnected_installation_mirroring/installing-mirroring-installation-images[Configuring a three-node cluster] in the {ocp-short} documentation for more details.
```
Comment (cadawson) [inline install/cluster_size.adoc:131]:
```suggestion
| See link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/disconnected_installation_mirroring/installing-mirroring-installation-images[Configuring a three-node cluster] in the {ocp-short} documentation for more details.
```

File: release_notes/fips_readiness.adoc [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 If you plan to manage clusters with FIPS enabled, you must install {acm-short} on an {ocp-short} cluster configured to operate in FIPS mode. The hub cluster must be in FIPS mode because cryptography that is created on the hub cluster is used on managed clusters. 
 
-To enable FIPS mode on your managed clusters, set `fips: true` when you provision your {ocp-short} managed cluster. You cannot enable FIPS after you provision your cluster. For more information, see {ocp-short} documentation, link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.14/html-single/installing/index#installing-preparing-security[Do you need extra security for your cluster?]
+To enable FIPS mode on your managed clusters, set `fips: true` when you provision your {ocp-short} managed cluster. You cannot enable FIPS after you provision your cluster. For more information, see {ocp-short} documentation, link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/installation_overview/installing-preparing#installing-preparing-security[Do you need extra security for your cluster?]
```
Comment (cadawson) [inline release_notes/fips_readiness.adoc:8]:
```suggestion
To enable FIPS mode on your managed clusters, set `fips: true` when you provision your {ocp-short} managed cluster. You cannot enable FIPS after you provision your cluster. For more information, see link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.14/html/installation_overview/installing-preparing#installing-preparing-security[Do you need extra security for your cluster?] in the {ocp-short} documentation.
```

**Review by cadawson — APPROVED** [review]
All of the links work. Made a couple of easy recommendations that I saw. Thanks!

**Comment by dhaiducek** [comment]
Hey @cadawson, I think those changes are fine, but they're semantic and present in `2.12_stage`. If those changes need to be made, I think they should be made in `2.12_stage` and cherry-picked--would you agree? Plus, I'd prefer these cherry-pick PRs to update the OCP URLs focus entirely on the links rather than any wording updates.

**Comment by dhaiducek** [comment]
@cadawson So it's not lost, I made a new PR to cherry-pick here:
- https://github.com/stolostron/rhacm-docs/pull/7034

**Comment by cadawson** [comment]
That sounds reasonable, @dhaiducek. I didn't realize that was the intent. It looks like you covered it, but we could have always just skipped them for now. Thanks for your help with the links! Much appreciated!

**Pull request merged**